### ISSUE TITLE
Please remove dot(.) in Request duration

### ISSUE THREAD

**Issue by colttt** [Graylog2/apache-mod_log_gelf#4]
Hello,

it would be nice if you remove the . in Request duration from 25.844 to 25844, i can use it with other apache-instances which dont support you modul for this i use:

```
LogFormat "{ \"version\": \"1.1\", \"host\": \"ringo.aei.mpg.de\", \"facility\": \"apache-gelf\", \"short_message\":  \"%r\", \"timestamp\": %{%s}t, \"level\": 6, \"_agent\":  \"%{User-Agent}i\", \"_bytes_send\": \"%O\", \"_request_duration_ms\": %D, \"_remote_address\": \"%a\", \"_request_method\": \"%m\", \"_referer\": \"%{Referer}i\", \"_status\": %s, \"_virtual_host\": \"%v\" }"  graylog2_access

CustomLog "|/bin/nc -u syslog.my.domain.de 50520" graylog2_access
```

so please remove the dot! or add a new GelfField (Request duration ms / Request duration seconds)

best regards

**Comment by mariussturm** [comment]
Sorry, I don't get the actual problem. Why should the dot be removed?
The current value is in milliseconds with 3 decimal digets. You can transform this value with log solutions like Graylog. So sending redundant informations should not be needed.

**Comment by mariussturm** [comment]
Ah, I think I got it now. You want to have the request_duration field be consistend with the normal LogFormat output.
I just released 0.2.0 of the module, the field is now renamed to `request_duration_us` to reflect the change to the user. The value is now in microseconds, like the %D output from mod_log_config.
